src/components/Lesson.vue

```diff
@@ -95,6 +98,12 @@
         </div>
       </div>
     </section>
+    <section class="indent-1 mb4 mw-900">
+      <div>
+        <p>Feeling stuck? We'd love to hear what's confusing so we can improve
+        this lesson. Please <a :href="issueUrl">share your questions and feedback</a>.</p>
```

Comment:
This would be more friendly if it opens github in a new tab, as we don't currently preserve your state across page reloads... right now if you click on it, your answer is lost.

```html
<a :href="issueUrl" target="_blank">
```